features/ALB010_Alignment-nesting-referents.feature

```diff
@@ -14,4 +14,4 @@ IfcReferent is associated to IfcAlignment via the IfcRelNests relationship.
       Given A model with Schema "IFC4.3"
       Given An IfcAlignment
       
-      Then A *required* relationship IfcRelNests from IfcAlignment to IfcReferent and following that
\ No newline at end of file
+      Then A relationship IfcRelNests exists from IfcAlignment to IfcReferent
```

Comment:
```suggestion
      Then A relationship IfcRelNests must exist from IfcAlignment to IfcReferent
```

Saying `the relationship exists` fits better as a Given.  Saying `must exist` fits better as a Then statement because it enforces a requirement to align with this industry best practice.

Reply:
[Implemented](https://github.com/buildingSMART/ifc-gherkin-rules/pull/294/commits/d4a5e1d8433c15b78ca3db31cb0538e644c96cc2)